src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -90,11 +90,17 @@ public void addNewInstructor(String name, String email) {
         waitForPageToLoad();
     }
 
-    public void editInstructor(int instrNum, String name, String email, String role) {
+    public void editInstructor(int instrNum, String name, String email,
```

Comment:
Fixed in https://github.com/TEAMMATES/teammates/pull/7559/commits/cf41d1694dc04925b4bf436fb698203a993d7f7a and https://github.com/TEAMMATES/teammates/pull/7559/commits/b120a0141e98a7ebd609dca9e3cee1333aff8447